strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/downloader/IndexDownloader.java

```diff
@@ -68,7 +69,7 @@ public void download(IndexDownloadRequest request)
                                                                           request.getRemoteRepositoryURL(),
                                                                           proxyRepositoryConnectionPoolConfigurationService.getHttpClient()));
 
-        updateRequest.setIndexTempDir(RepositoryFiles.temporary(repositoryPathResolver.resolve(request.getRepository())).toFile());
+        updateRequest.setIndexTempDir(new File(RepositoryFiles.temporary(repositoryPathResolver.resolve(request.getRepository())).toUri().getPath()));
```

Comment:
@fuss86 / @sbespalov : Is this in line with the way things should be done with the file-system/path approach that we must follow?